evm/deploy-amplifier-gateway.js

```diff
@@ -184,7 +184,7 @@ async function deploy(config, chain, options) {
     contractConfig.deployer = wallet.address;
     const domainSeparator = await getDomainSeparator(config, chain, options);
     const minimumRotationDelay = Number(options.minimumRotationDelay);
-    const salt = options.salt || '';
+    const salt = options.salt || `AxelarAmplifierGateway ${config.axelar.chainId}`;
```

Comment:
```suggestion
    const salt = options.salt || 'AxelarAmplifierGateway';
```
Let's use this as the default, so testnet/mainnet derive the same addresses. For devnets, our orchestration can pass in an appropriate salt perhaps